### PR TITLE
Update translations for upcoming 18.4

### PR DESCRIPTION
**Pull request by def-**
## Checklist

- [ ] Tested the change ingame
- [ ] Provided screenshots if it is a visual change
- [ ] Tested in combination with possibly related configuration options
- [ ] Written a unit test (especially base/) or added coverage to integration test
- [ ] Considered possible null pointers and out of bounds array indexing
- [ ] Changed no physics that affect existing maps
- [ ] Tested the change with [ASan+UBSan or valgrind's memcheck](https://github.com/ddnet/ddnet/#using-addresssanitizer--undefinedbehavioursanitizer-or-valgrinds-memcheck) (optional)

### CODE REVIEW

File: data/languages/german.txt [hunk 1]
```diff
@@ -1868,3 +1859,27 @@ Tee
...
+[Spectators]
+%d others…
+== %d andere…
+
+[Team and size]
+%d\n(%d/%d)
+== %d\n(%d/%d)
+
+Team %d (%d/%d)
+== Team %d(%d/%d)
```
Comment (Robyt3) [inline data/languages/german.txt:1882]: Space before `(`

File: data/languages/german.txt [hunk 2]
```diff
@@ -1868,3 +1859,27 @@ Tee
...
+
+https://wiki.ddnet.org/wiki/Mapping
+==
```
Comment (Robyt3) [inline data/languages/german.txt:1885]: Missing space after `==` is likely causing the failed check

**Review by Robyt3 — CHANGES_REQUESTED** [review]

File: data/languages/arabic.txt [hunk 3]
```diff
@@ -296,9 +293,6 @@ Reason:
 Red team
 == ﺮﻤﺣﻻﺍ ﻖﻳﺮﻔﻟﺍ
 
-Red team wins!
```
Comment (heinrich5991) [inline data/languages/arabic.txt:299]: Why did these strings go away?

Comment (Robyt3) [reply data/languages/arabic.txt:299]: They are now covered by the existing `%s wins!` where `%s` is the clan or localized name of the team.

Comment (Robyt3) [reply data/languages/arabic.txt:299]: See #8600.

**Review by Robyt3 — APPROVED** [review]

**Pull request merged**